pages/linux/makepkg.md

```diff
@@ -2,7 +2,7 @@
 
 > Creates a package installable with the `pacman` package manager.
 > Runs the commands from a `PKGBUILD` file to build the package.
-> More information: <https://wiki.archlinux.org/index.php/Makepkg>.
+> More information: <https://man.archlinux.org/man/makepkg.8>.
```

Comment:
```suggestion
> More information: <https://man.archlinux.org/man/makepkg.8>.
```

It redirects however you can remove the `.8` from the end.

Reply:
Check `grep -Rn 'man\.archlinux'` we always use the chapter suffix for Arch Linux's man pages.